### DISCUSSION TITLE
How to installer postgresql and pgadmin4 in laragon

### DISCUSSION THREAD

**Discussion by apkeven** [leokhoa/laragon#503] [General]
How to installer postgresql and pgadmin4 in laragon ?

**Comment by Pen-y-Fan** [comment]
PostgreSQL can be installed using quick add

![image](https://github.com/leokhoa/laragon/assets/40126936/e771bbd5-f48c-4c14-8f7b-ced70306829c)

Info on pgadmin: https://github.com/leokhoa/laragon/issues/345

**Reply by Pen-y-Fan** [reply]
To add PostgreSQL 5 see https://github.com/leokhoa/laragon/discussions/386
